main.cpp

```diff
@@ -108,7 +108,44 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Person
+{	
+    int age;	    
+    int height;	    
+    float hairLength;	    
+    float GPA;
+    unsigned int SATScore;
+    int distanceTravelled;
 
+    void run(int, bool);
+
+};
+struct Foot
+{
+    int stepSize();
+    void stepForward();
+};
+int Foot::stepSize()
+{
+    return 8;
+}
+
+void Foot::stepForward()
+{
+
+}
+
+void Person::run(int howFast, bool startWithLeftFoot)
+{
+    if (startWithLeftFoot)
+    {
+        distanceTravelled += leftFoot.stepSize + rightFoot.stepSize;
+    }
+    else 
+    {
+        distanceTravelled += rightFoot.stepSize + leftFoot.stepSize;
+    }
```

Comment:
You can't use instances of `Foot` unless you declare them as class members in `Person.
- [ ] revise